tests/memex/conftest.py

```diff
@@ -17,6 +17,7 @@
 from sqlalchemy.orm import sessionmaker
 
 from memex import db
+from memex import models  # noqa: ensure that model base class is set
```

Comment:
This is here so that if you run a single memex test module that *doesn't* depend on models, you can still use the database when the `db_engine` fixture calls `init`. `query_test.py`, added in this commit, is an example of a test module which uses the database but not the models.